config/environments/development.rb

```diff
@@ -1,17 +1,15 @@
 Rails.application.configure do
-  # Settings specified here will take precedence over those in config/application.rb.
-
-  # In the development environment your application's code is reloaded on
-  # every request. This slows down response time but is perfect for development
-  # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
-
-  # Do not eager load code on boot.
-  config.eager_load = false
-
-  # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
-  config.action_controller.perform_caching = false
+  # Set up production configuration
+  config.eager_load = true
+  config.cache_classes = true
+  config.action_controller.perform_caching = true
+  config.action_controller.cache_store = :dalli_store
+  config.consider_all_requests_local = false
+  # Configure logging
+  ActiveSupport::Cache::Store.logger = config.logger
+  config.log_level = :debug
+  # otherwise deadlocks can occurr
+  config.middleware.delete 'Rack::Lock'
```

Comment:
Not sure what env you were running in; I assume dev. So this, makes it behave like prod.